### ISSUE TITLE
Extremely Weird Sand Generation on top of Bedrock...

### ISSUE THREAD

**Issue by Jusey1** [yungnickyoung/Better-Caves#145]
I installed the mod recently and everything looks great. So I changed a config to set the water at Y 10 and below to be 100% instead of 40% and now all of a sudden, I am getting this weird generation where sand blocks are randomly generating right above bedrock...

![image](https://user-images.githubusercontent.com/61861078/139522971-35273535-2c89-4be7-9dd9-3a27a26f887c.png)

Removing the mod is the only way to fix the problem. So... What gives? Why is this happening?

**Comment by yungnickyoung** [comment]
I'm not sure. I think this is because I'm using the biome's filler block when replacing bedrock. Was this under a desert?

**Comment by Jusey1** [comment]
Nope. This happens EVERYWHERE and only when Flat-Bedrock is enabled.

**Comment by yungnickyoung** [comment]
What version is this? And are you using other worldgen mods?

**Comment by Jusey1** [comment]
1.1.2 for Forge 36.2.0 on Minecraft 1.16.5. I tested this with just the mod by itself and it still happens and only when flat bedrock is enabled.
